bodhi/client/__init__.py

```diff
@@ -456,7 +456,14 @@ def save_buildroot_overrides(user, password, url, staging, **kwargs):
         staging (bool): Whether to use the staging server or not.
         kwargs (dict): Other keyword arguments passed to us by click.
     """
-    _save_override(url=url, user=user, password=password, staging=staging, **kwargs)
+
+    try:
+        _save_override(url=url, user=user, password=password, staging=staging, **kwargs)
+    except bindings.BodhiClientException as e:
+        click.echo(str(e))
+        if str(e) == "Buildroot override for %s already exists" % (kwargs['nvr']):
+            click.echo("The `overrides save` command is used for creating a new override")
+            click.echo("Use `overrides edit` to edit an existing override")
```

Comment:
I would suggest moving the ```click.echo()``` into the ```if``` block, and adding an ```else``` with ```raise```, so other errors still print a traceback. That way we can get better debugging info if users experience a different problem than this one.